### ISSUE TITLE
Can't get it working with AAD

### ISSUE THREAD

**Issue by frost-cqre** [SAPUCC/inviterbot#9]
Hi,
I am trying to manage users and rooms with AAD, but I can't get over this error:

```
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/mautrix/client/syncer.py", line 235, in _catch_errors
    await handler(data)
  File "/opt/maubot/maubot/handlers/command.py", line 154, in __call__
    return await self.__mb_func__(self.__bound_instance__, evt, **call_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/plugins/de.in4md-service.inviterbot-v0.1.4.mbp/inviter/bot.py", line 109, in idp
    room_struct = await helper.get_room_struct_from_idp(self.config)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/plugins/de.in4md-service.inviterbot-v0.1.4.mbp/inviter/bot_helper.py", line 65, in get_room_struct_from_idp
    room_struct = await ms_graph_connector.get_rooms(config)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/plugins/de.in4md-service.inviterbot-v0.1.4.mbp/inviter/ms_graph_connector.py", line 143, in get_rooms
    mxid = config.get_renamed_mxid(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/plugins/de.in4md-service.inviterbot-v0.1.4.mbp/inviter/config.py", line 47, in get_renamed_mxid
    if renamed_users.get(mxid.username):
       ^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
```

It's in the logs every time I try to do anything that would connect to AAD. I suspect there is something wrong with my AAD user groups setup, but I tried to recreate it as much exactly as possible according to documentation.

Is there some more info that I can look at to further troubleshoot the issue?

Thanks.

**Comment by v411e** [comment]
Hi, thank you for filing this issue. You did nothing wrong. It looks like I forgot to properly handle the case when there are no users in the renamed users config. This config option is used for the rare case, that a person changes their name in AAD, because then we want the bot to still use the old matrix handle as the matrix user name does not change.

Will have a look at this issue :)

**Comment by v411e** [comment]
Fixed by #10
